### DISCUSSION TITLE
How is this different from DamianOsipiuk/vue-query ?

### DISCUSSION THREAD

**Discussion by MrSunshyne** [TanStack/query#3890] [General] [answered]
After reading the docs a little, the api appears to be very similar to DamianOsipiuk/vue-query which itself is inspired by react-query. 

IIRC all of them use the following approach : 
```js
import {
  useQuery,
  useMutation,
} from 'my-query-library'

const query = useQuery(['todos'], getTodos)
```

Will/Does TanStack/query offer different features ?

If anything, having multiple library implement very similar(identical?) patterns suggests that this is the right way of handling client site queries :partying_face:

**Comment by TkDodo** [comment, marked as answer]
`DamianOsipiuk/vue-query` is based on react-query - it uses the v3 query core internally. The plan is that vue-query will _become_ `@TanStack/vue-query`. So we'll likely integrate the code into the TanStack monorepo and @DamianOsipiuk will become the maintainer of it.

**Reply by MrSunshyne** [reply]
> The plan is that vue-query will become @TanStack/vue-query

That's actually awesome. Thanks for clarifying!
